installing/install_config/customizations.adoc

```diff
@@ -5,91 +5,175 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 You complete most of the cluster configuration and customization after you
-deploy your {product-title} cluster. By default, a number of
+deploy your {product-title} cluster. A number of
 _configuration resources_
 //and _Custom Resources_
 are available.
 
-You modify the configuration resources to define the major components of the
-cluster, such as the networking configuration and the identity provider.
+You modify the configuration resources to configure the major components of the
+cluster, such as the image registry, networking configuration, image build 
+behavior, and the identity provider.
 
+For current documentation of settings these resources expose, use
+the `oc explain` command, for example `oc explain builds --api-version=config.openshift.io/v1`
 ////
 You can also deploy Custom Resources (CRs) that are based on a number of
 Custom Resource Definitions (CRDs) that are deployed on your cluster.
 ////
 
 [id="configuration-resources_{context}"]
-== Configuration resources
+== Cluster configuration resources
+
+All cluster configuration resources are globally scoped (not namespaced) and named `cluster`.
```

Comment:
can you add a comment (not visible in rendered docs I hope), that say that if you find yourself writing a help page that requires coordinated changes on two resources, they should tag api-approvers and get some help rethinking their API?

Reply:
i.....can......  but i would imagine by the time someone is writing that doc the ship has already sailed.

your comment makes me think this has already happened? :)

Reply:
comment added.